app/jobs/export_digital_service_accounts.rb

```diff
@@ -1,29 +1,22 @@
 # frozen_string_literal: true
 
 class ExportDigitalServiceAccounts < ApplicationJob
-  include ApplicationHelper
+  include S3Helper
 
   queue_as :default
 
-  def perform(session_uuid, include_all_accounts: false, filename: "touchpoints-digital-service-accounts-#{Time.now.to_i}.csv")
```

Comment:
When I run this locally, this job fails with `ArgumentError: unknown keyword: :filename`.

Reply:
same. writing spec.